server/Sample/sfreerdp.c

```diff
@@ -918,6 +919,7 @@ int main(int argc, char* argv[])
 	}
 
 	WTSRegisterWtsApiFunctionTable(FreeRDP_InitWtsApi());
+	winpr_InitializeSSL(WINPR_SSL_INIT_DEFAULT);
```

Comment:
Not good, better move this in the ```*_new``` functions, client/server code should not require such initialisation stuff.